components/badge/badge.c

```diff
@@ -37,8 +37,7 @@ touch_event_handler(int event)
 			int id = conv[(event >> 8) & 0xff];
 			if (id != -1)
 			{
-				uint32_t button_down = id;
-				xQueueSend(badge_input_queue, &button_down, 0);
+				badge_input_add_event(id, true, false);
```

Comment:
added EVENT_BUTTON_PRESSED (to avoid confusion with BUTTON_DOWN) / EVENT_BUTTON_RELEASED / IN_ISR / NOT_IN_ISR